### PR TITLE
rev to v2020.9.0-preview and update CL

### PR DESCRIPTION
**Pull request by TylerLeonhardt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
 # PowerShell Extension Release History
 
+## v2020.9.0-preview
+### Wednesday, September 16, 2020
+#### [vscode-PowerShell](https://github.com/PowerShell/vscode-PowerShell)
+
+- 📺✨ [vscode-powershell #2919](https://github.com/PowerShell/vscode-powershell/pull/2919) -
+  Untitled Notebooks and Automatic Notebooks.
```
Comment (rjmholt) [inline CHANGELOG.md:8]: Could improve this entry

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+- 👷✨ [vscode-powershell #2881](https://github.com/PowerShell/vscode-powershell/pull/2881) -
+  Remove UTF-8 BOM from misc text files. (Thanks @xtqqczze!)
+- 👷✨ [vscode-powershell #2876](https://github.com/PowerShell/vscode-powershell/pull/2876) -
+  Update text in LICENSE.txt. (Thanks @xtqqczze!)
+- 📺✨ [vscode-powershell #2861](https://github.com/PowerShell/vscode-powershell/pull/2861) -
+  Update LSP Client to use new Semantic Highlighting APIs. (Thanks @justinytchen!)
+- 👷✨ [vscode-powershell #2871](https://github.com/PowerShell/vscode-powershell/pull/2871) -
+  Prevent `Update Notebook dts` action from triggering on forked repositories. (Thanks @corbob!)
+- 🔧🐛 [vscode-powershell #2863](https://github.com/PowerShell/vscode-powershell/pull/2863) -
+  Fix migration of `codeFormatting.addWhitespaceAroundPipe` setting.
```
Comment (rjmholt) [inline CHANGELOG.md:28]: Maybe worth mentioning that this won't add this setting when it doesn't already exist now

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,51 @@
...
+- 📺🐛 [vscode-powershell #2845](https://github.com/PowerShell/vscode-powershell/pull/2848) -
+  Handle block comments with text on the same line as <# #> in Notebook Mode.
+- ✨ [vscode-powershell #2855](https://github.com/PowerShell/vscode-powershell/pull/2855) -
+  Expose the API via exports instead of editor commands.
```
Comment (rjmholt) [inline CHANGELOG.md:32]:
```suggestion
  Expose the notebook API via exports instead of editor commands.
```

**Review by rjmholt — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (TylerLeonhardt) [reply CHANGELOG.md:28]:
```suggestion
  Fix migration of `codeFormatting.addWhitespaceAroundPipe` setting when it doesn't already exist.
```

File: CHANGELOG.md (hunk 1 above)
Comment (TylerLeonhardt) [reply CHANGELOG.md:8]:
```suggestion
  Create new Untitled Notebooks and support for Automatic Notebooks (`*.Notebook.ps1`).
```

**Pull request merged**